src/containers/Home.jsx

```diff
@@ -87,10 +87,10 @@ const Content = (props) => {
       <Row>
         <Col sm={7} className="mx-3 mx-sm-0 blurb-content">
           <div>
-            <h3>Curious about life in SE? Check out our latest feature: </h3>
+            <h3>Want to stay connected with the SE community? Join our Discord: </h3>
             <p>
-              <a href={featureLink} target="_blank" rel="noopener noreferrer">
-                {recentBlogPost.title}
```

Comment:
nit: remove line 8: `import { recentBlogPost } from 'content/BlogContent'` and line 45-47: `featureLink` to fix the linter error.